spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -387,7 +387,9 @@ def step(  # noqa
         if self.prev_base_moved and base_action is None:
             self.spot.stand()
 
-        print(f"base_action: {arr2str(base_action)}\tarm_action: {arr2str(arm_action)}")
+        self.say(
+            f"base_action: {arr2str(base_action)}\tarm_action: {arr2str(arm_action)}"
+        )
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.